File: src/main/java/jp/otknoy/av/dmm/item/Response.java
package jp.otknoy.av.dmm.item;

import lombok.Builder;
import lombok.Data;

import java.io.Serializable;

@Data
@Builder
public class Response implements Serializable {
    private int resultCount;
    private int totalCount;
    private int firstPosition;
    private Result result;
}

File: response.go
package main

type Item struct {
	Title    string `json:"title"`
	Url      string `json:"url"`
	ImageUrl string `json:"image_url"`
}

type Items []Item

type Response struct {
	ResultCount   int64 `json:"result_count"`
	TotalCount    int64 `json:"total_count"`
	FirstPosition int64 `json:"first_position"`
	Items         Items `json:"items"`
}
